### PR TITLE
Conceptual topics, glossary updates

### PR DESCRIPTION
**Pull request by dg-concordium**
## Purpose

Updates to make the conceptual topics clearer and more complete.

## Changes

Added table to transactions topic to show where various transactions can be executed. The text below the table (in a bulleted list) will go away once the table contents are verified.
Redid the structure of the sidebar to make it more compact.
Added Backups topic.
Split Identities and Accounts topic into two topics.
Added illustrations to some conceptual topics to make them more clear.

## Checklist

- [ X] My code follows the style of this project.
- [ X] The code compiles without warnings.
- [ X] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

_Remove if not applicable.

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [ X] I accept the above linked CLA.

### CODE REVIEW

File: source/shared/net/concepts/concepts-baker.rst [hunk 1]
```diff
@@ -11,16 +11,18 @@ How baking works
 
 A node is a baker node when it participates actively in the network by creating new blocks that are added to the chain. A baker collects, orders, and validates the transactions that are included in a block to maintain the integrity of the blockchain. The bakers sign each block that they bake so that the block can be verified and executed by the rest of the participants of the network.
 
+The chain a baker builds upon the :ref:`best chain <glossary-best-chain>` when making a new block. The best chain is selected using :ref:`consensus protocol <glossary-consensus>`. In particular, the best chain has the most finalized blocks and the most blocks after the last finalized block.
```
Comment (eb-concordium) [inline source/shared/net/concepts/concepts-baker.rst:14]: Should it have been `The chain a baker builds upon *is* the ...`?

File: source/shared/net/concepts/concepts-baker.rst [hunk 2]
```diff
@@ -36,15 +38,40 @@ A baker needs to stake a part of its CCD balance on the baker account. Later, th
...
+   :alt: timeline showing how 14400 slots make up one epoch
+
+When considering the rewards and other baking-related concepts, the concept of an *epoch* is used as a unit of time that defines a period in which the set of current bakers and stakes are fixed. Epochs have a duration of 1 hour and the duration is fixed at the :ref:`Genesis block <glossary-genesis-block>`.
+
+Epochs are subdivided into slots. On any given :ref:`branch <glossary-branch>`, each slot can have a maximum of one block, but multiple blocks on different branches can be produced in the same slot. Slots have a duration of 250ms, and the duration is fixed at the :ref:`Genesis block <glossary-genesis-block>`.
+
+A :ref:`cool-down period <glossary-cool-down-period>` describes a period of time during which certain activities or transactions are frozen. For example, if you decrease a baker stake in the desktop wallet, the stake will be decreased after a cool-down period. The cool-down period is 2 epochs. During the cool-down period, you’ll not be able update the stake. After the cool-down period, the amount by which you decreased your stake is returned to your disposable balance.
```
Comment (eb-concordium) [inline source/shared/net/concepts/concepts-baker.rst:57]: Isn't the cool-down period currently `166 epochs ~ 1 week` on Mainnet?

File: source/shared/net/concepts/id-accounts.rst [hunk 3]
```diff
@@ -1,115 +1,86 @@
...
-entities learn any information about the account owner. No information is
-publicly revealed.
+Identity issuance requires *Identity Verification*, which is the process of verifying the real-life identity of the user. This typically requires taking photographs or scans of identification documents, such as a passport. Identity verification also checks that the user-chosen attributes are valid for the user.
+
+Upon verification of the user's identification documents and attributes, the Identity provider issues a :ref:`user identity certificate<glossary-user-identity-certificate>`. The User identity certificate contains attributes about the user. It is basically the Identity Provider’s signature over some cryptographic keys of the user and the validated personal attributes.
+
+.. image:: ../images/concepts/identity-creation.png
+   :alt: graphic drawing showing how the user interacts with the identity provider
+
+About accounts
```
Comment (eb-concordium) [inline source/shared/net/concepts/id-accounts.rst:56]: This section seems to be almost identical to what is on the Accounts page. Is this intended?

File: source/shared/net/concepts/concepts-baker.rst (hunk 2 above)
Comment (dg-concordium) [reply source/shared/net/concepts/concepts-baker.rst:57]: According to Thomas it varies based on the transaction.

File: source/shared/net/concepts/id-accounts.rst (hunk 3 above)
Comment (dg-concordium) [reply source/shared/net/concepts/id-accounts.rst:56]: The two drawings should be different. One should show purely identity flow and one should show how the user can create accounts.

File: source/shared/net/concepts/concepts-transactions.rst [hunk 4]
```diff
@@ -10,22 +10,36 @@ Transactions overview
...
+The most basic transaction is the CCD transfer that is used to send CCD from one account to another. However, there are several transaction types on the Concordium blockchain.
 
 You can make transactions using either the Desktop Wallet, the Mobile Wallet, or the Concordium Client. Note that the Mobile Wallet doesn’t support all transaction types, and that you need a Ledger device to submit transactions from the Desktop Wallet.
 
++----------------------------+---------------+----------------+-------------------+
+|                            | Mobile Wallet | Desktop Wallet | Concordium Client |
++============================+===============+================+===================+
+| Send CCD                   | |check|       | |check|        | |check|           |
++----------------------------+---------------+----------------+-------------------+
+| Send CCD with a schedule   | |check|       | |check|        | |check|           |
```
Comment (eb-concordium) [inline source/shared/net/concepts/concepts-transactions.rst:22]: I think maybe this `|check|` under Mobile Wallet should be removed.

**Review by eb-concordium — COMMENTED** [review]
Looks good. I only have a few minor comments.

**Review by eb-concordium — APPROVED** [review]

**Pull request merged**